fc/jira/task.py

```diff
@@ -81,7 +78,30 @@ def _extra_json_for_create(self, existing_json: dict):
     def type_str(self) -> str:
         raise NotImplementedError
 
-    def _transition(self, id_of_transition: str):
+    def transition(self, state: str):
+
+        # use created Task (could be Backlog task or Triage task) to transition to desired state
+        # Look up starting state in dictionary
+        # then look up task type (Backlog or Triage)
+        # then look up end state
+        # array is the sequence of state transitions in order to iterate through
+
+        transition_dict = self._get_transition_dict()
+
+        transition_arr = transition_dict[self.state][state]
```

Comment:
We should perhaps wrap this statement in a `try`/`except` block to catch a `KeyError` and `TypeError`.  First one if there isn't an appropriate key in the transition dictionary.  Second one if there is the key, but it's `None`; aka, if `transition_dict[self.state]` returns `None` (a `TypeError` is raised if `None[state]` happens).